doc/whatsnew/2.7.rst

```diff
@@ -21,3 +21,7 @@ Other Changes
 * Fix bug that lead to duplicate messages when using ``--jobs 2`` or more.
 
   Close #3584
+
+* Enums are now required to be named in UPPER_CASE by ``invalid-name``.
+
+  Close #3834
```

Comment:
Usually we do not refer to issue in the whatsnew doc. By the way if you could remove the one above it would be nice.

Reply:
fixed at 31965a1